cdk/visit/__init__.py

```diff
@@ -44,13 +44,16 @@ def __init__(self, scope: core.Construct,
 
         self.source_bucket()
 
+        # Retrieves the domain and its corresponding stage
+        self.domain_stage = self.zones.domains.visit
```

Comment:
`domain_stage` feels pretty confusing to me. Can you rename this to something like `api_endpoint`, `api_base_url` or `api_url`?

The prod api base url `api.cuMaker.space` doesn't even have the stage in it.

Reply:
The way I implemented it, every dev stack would be using the dev-visit.cumaker.space, I had just assumed that the variable should correspond with the stage that was passed into the lambda functions, but it would make sense to have it use dev-{user}-visit.cumaker.space for the individual dev accounts to use. I can go ahead and work on changing it to use that instead.